PiwikTracker/PiwikTracker.swift

```diff
@@ -128,6 +128,14 @@ final public class PiwikTracker: NSObject {
     internal var visitor = Visitor.current()
     internal var session = Session.current()
     internal var nextEventStartsANewSession = true
+    
+    public func userAgent () -> String {
+        return self.dispatcher.userAgent!
+    }
+    public func setUserAgent (userAgent: String)
+    {
+        self.dispatcher.setUserAgent(ua: userAgent)
+    }
```

Comment:
I will keep the mutable `userAgent` property and also add a optional parameter in the init function.
I don't want to create default user agent everywhere.